StRoot/StETofCalibMaker/StETofCalibMaker.cxx

```diff
@@ -1343,11 +1360,11 @@ StETofCalibMaker::processMuDst()
 	}*/
 
     size_t nDigis = mMuDst->numberOfETofDigi();
-    //LOG_INFO << "processMuDst() - # fired eTOF digis : " << nDigis << endm;
+    LOG_INFO << "processMuDst() - # fired eTOF digis : " << nDigis << endm;
```

Comment:
You uncommented or added a bunch of printout statements. Are you suggesting to print all of them for every event? Our log files are already bloated. Preferably only warnings and errors should be logged.

Reply:
No, i accidently pushed the wrong commit. All the debugging messages should be removed now. This exact line that you commented on though has historically always been in.

Reply:
I probably already deactived this line in a former print-out clean-up. "Historically", the idea was to check that the same number of data members are seen by different makers. This information could not be obtained for single events from the output file. On the other hand, probably nobody is ever going through production logs and check for single event level issues. That is a case for local debugging. If bloated log files are becoming a problem, then all event-by-event print-outs should go. I removed them from all relevant eTOF makers in the latest commit unless they are hidden behind doQA flags or debug flags.